projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -549,13 +583,6 @@ on production builds, the attributes are being reordered, causing side-effects
 	padding-right: var(--field-padding);
 }
 
-.contact-form .is-style-animated .grunion-field-checkbox-multiple-wrap .grunion-checkbox-multiple-options,
-.contact-form .is-style-animated .grunion-field-radio-wrap .grunion-radio-options {
```

Comment:
The _Animated_ style only add some odd padding at the moment. Let's remove it altogether for now and rework the style later on if needed.